### ISSUE TITLE
Feature Request - refresh individual slot

### ISSUE THREAD

**Issue by bigwillch** [jaanauati/react-dfp#102]
It would be really useful if we could refresh individual slots using `DFPManager.refresh([slotId])` in the same way as can be done with `DFPManager.load()`

I have tried to achieve this by using googletag directly like so, however have not had any luck (maybe I'm missing something here?)

```js
const slotId = 'adSlot-1';

DFPManager.getGoogletag().then((googletag) => {
  googletag.cmd.push(() => {
    googletag.pubads().refresh(DFPManager.getRefreshableSlots()[slotId].gptSlot);
  });
});
```

**Comment by jaanauati** [comment]
hey, that feature makes sense!..., on the other hand, have you tried with the prop `shouldRefresh`?

Here an example, notice that only the slot 'test/refreshable' is getting refreshed (see details in the doc):
```
import React, { Component } from 'react';
import ReactDom from 'react-dom';
import { AdSlot, DFPManager, DFPSlotsProvider } from 'react-dfp';

export default class RefreshableAdsExample extends Component {

    componentDidMount() {
        this.interval = window.setInterval(
            () => { DFPManager.refresh(); },
            15000,
        );
    }

    componentWillUnmount() {
        window.clearInterval(this.interval);
    }

    render() {
        return (
            <div>
                <DFPSlotsProvider dfpNetworkId="9999">
                    <div>
                        <AdSlot
                            sizes={[[728,90], [300, 250]]}
                            adUnit='test/non-refreshable'
                            shouldRefresh={ () => false }
                        />
                    </div>
                    <div> </div>
                    <AdSlot sizes={[[300, 250]]} adUnit='test/refreshable' />
                </DFPSlotsProvider>
            </div>
        );
    }
}
```
thanks.

**Comment by bigwillch** [comment]
ah, I hadn't considered that approach. Will have a play, thanks!

**Comment by bigwillch** [comment]
Thanks for the above. I'm not sure it would help in our use case. We'd need it to be possible for any slot to be refreshable, just to have control over when that happens (I'd like to be able to use the sizemapping for each adslot to trigger the relevant adslot to refresh when the window is resized)

I guess I might be able to use `forceUpdate()`, but it would be great if the functionality could be provided (I'll have a stab at a PR if I can)

**Comment by bigwillch** [comment]
Have started working on an approach. Will create a PR when ready

https://github.com/bigwillch/react-dfp/tree/feature/refresh-individual-slots

**Comment by jaanauati** [comment]
merged and published! see v0.9.2.

Thank you!
